jupyterbook/_config.yml

```diff
@@ -40,13 +40,21 @@ parse:
 #######################################################################################
 # HTML-specific settings
 html:
-  favicon                   : "images/IOOS-logo_little.png"  # A path to a favicon image
+  favicon                   : "images/IOOS_logo.png"  # A path to a favicon image
   use_edit_page_button      : false  # Whether to add an "edit this page" button to pages. If `true`, repository information in repository: must be filled in
   use_repository_button     : false  # Whether to add a link to your repository button
   use_issues_button         : true  # Whether to add an "open an issue" button
```

Comment:
Any way we can use the [existing template](https://github.com/ioos/notebooks_demos/blob/master/.github/ISSUE_TEMPLATE.md)?

Reply:
I'll check.